src/coverage.ts

```diff
@@ -67,9 +77,17 @@ export const coverage = async ({
         description: `${percentString(rate)} coverage`
       }
     }
+    const state = evaluateState([
+      rate * 100 >= threshold,
+      Math.abs(diff * 100) >= differenceThreshold // absolute value because it is diff
+    ])
+    const failDescription =
+      state === 'success'
+        ? ''
+        : `Failed to satisfy threshold:${threshold}% and difference-threshold:${differenceThreshold}%`
     return {
-      state: diff < -0.0001 ? 'failure' : 'success',
-      description: `${percentString(rate)} (${percentString(diff)}) compared to ${compareSha.slice(0, 8)}`
+      state,
+      description: `${percentString(rate)} (${percentString(diff)}) compared to ${compareSha.slice(0, 8)}.${failDescription}`
```

Comment:
Won't this render as `... compared to abcdefg.Failed to satisfy threshold ...` (without a space after `.`)?

Reply:
fixed.